src/twister2/yaml_test_specification.py

```diff
@@ -55,3 +64,101 @@ def __post_init__(self):
         self.extra_args = string_to_list(self.extra_args)
         self.integration_platforms = string_to_list(self.integration_platforms)
         self.timeout *= self.timeout_multiplier
+
+
+# Using marshmallow to validate specification from yaml
+class HarnessConfigSchema(Schema):
+    type = fields.Str()
+    fixture = fields.Str()
+    ordered = fields.Str()
+    repeat = fields.Str()
+    pytest_root = fields.Str()
+    pytest_args = fields.List(fields.Str())
+    regex = fields.List(fields.Str())
+    record = fields.Dict(fields.Str(), fields.Str())
+
+
+class CommonSchema(Schema):
```

Comment:
`CommonSchema` contains exactly the same attributes as `TestsSchema`. Is there possibility to reuse somehow `TestsSchema` in `CommonSchema`? Something like:
```python
class CommonSchema(TestsSchema)
    pass
```
I ask about this because in Twister v1 this is quite painful when you make any change into `tests` schema you have to remember to update also `common` schema. Maybe there in v2 we could avoid it?

Reply:
Changed `CommonSchema` to inherit from `TestsSchema`.